internal/image/image_installer.go

```diff
@@ -130,6 +130,9 @@ func (img *ImageInstaller) InstantiateManifest(m *manifest.Manifest,
 	osPipeline.Environment = img.Environment
 	osPipeline.Workload = img.Workload
 
+	// enable ISOLinux on x86_64 only
+	isoLinux := img.Platform.GetArch() == platform.ARCH_X86_64
```

Comment:
Might be clearer to label this something like `isoLinuxEnabled`

Reply:
this is fixed